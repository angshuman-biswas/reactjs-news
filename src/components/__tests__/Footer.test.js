import { render } from '@testing-library/react';
import React from 'react';
import Footer from '../core/Footer';

test('<Footer/> Renders correctly', () => {
	const { asFragment } = render(<Footer />);
	expect(asFragment()).toMatchSnapshot();
});
