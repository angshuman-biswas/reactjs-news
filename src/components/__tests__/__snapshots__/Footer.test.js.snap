// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> Renders correctly 1`] = `
<DocumentFragment>
  <footer>
    <hr />
    Developed by
    <a
      href="https://arunabharjun.com"
      target="_blank"
    >
       Arunabh Arjun
    </a>
  </footer>
</DocumentFragment>
`;
